index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <nav id="nav-bar"></nav>
+        <img id="header-img" src="Images/fruits..jpeg" alt=" This is a picture of fruits" />
+        <nav id="nav-bar" class="sticky">
+            <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how_it_works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>
+        </nav>
```

Comment:
Please use HTML semantic tags. Some benefits of using these tags are:
.Provide your page with a search engine optimization score
.Easy code navigation when updating features or troubleshooting bugs.